eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -160,10 +161,11 @@ jobs:
             ${{ pair.key }}: ${{ pair.value }}
       - template: /eng/pipelines/templates/steps/install-dotnet.yml
       - script: >-
-          dotnet test eng/service.proj --filter TestCategory!=Live --framework $(TestTargetFramework)
+          dotnet test eng/service.proj --filter "(TestCategory!=Manually) & (TestCategory!=Live)" --framework $(TestTargetFramework)
```

Comment:
The `Manually` filter existed in the tests archtype; it seemed like it should be applied here as well.